pkgs/by-name/ne/neovim-node-client/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchNpmDeps,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "node-client";
+  version = "5.3.0";
+
+  src = fetchFromGitHub {
+    owner = "neovim";
+    repo = "node-client";
+    rev = "02a5fdef8e3ec2812d295981db38dbccf82e0728";
+    hash = "sha256-0vPw2hCGUDepSpF1gp/lI71EgwGsCSnw7ePP7ElHsTQ=";
+  };
+
+  npmDeps = fetchNpmDeps {
+    inherit src;
+    hash = "sha256-VYoJAi1RzVf5ObjuGmnuiA/1WYBWC+qYPdfWF98+oGw=";
+  };
+  npmWorkspace = "packages/neovim";
+
+  postInstall = ''
+    mkdir -p $out/bin
+    # Overwrite the unwanted wrapper created by buildNpmPackage
+    ln -sf $out/lib/node_modules/neovim/bin/cli.js $out/bin/neovim-node-host
+  '';
+
+  meta = {
+    mainProgram = "node-client";
```

Comment:
```suggestion
    mainProgram = "neovim-node-host";
```

Reply:
Well, this PR was merged a few days ago, so you would probably want to file a fixup PR

Reply:
I won't do that. The author of this PR or the reviewers should.

Reply:
sha256-VYoJAi1RzVf5ObjuGmnuiA/1WYBWC+qYPdfWF98+oGw=